example/laravel56/install_composer.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+EXPECTED_SIGNATURE="$(wget -q -O - https://composer.github.io/installer.sig)"
+php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+ACTUAL_SIGNATURE="$(php -r "echo hash_file('SHA384', 'composer-setup.php');")"
+
+if [ "$EXPECTED_SIGNATURE" != "$ACTUAL_SIGNATURE" ]
+then
+    >&2 echo 'ERROR: Invalid installer signature'
+    rm composer-setup.php
+    exit 1
+fi
+
+php composer-setup.php
+RESULT=$?
+rm composer-setup.php
+exit $RESULT
```

Comment:
would it be better to just use the composer image directly so we don't need this script?

Reply:
The composer image only runs composer, and isn't designed for running PHP servers unfortunately.